docs/content/configuration/creating_volumestrorageclass.md

```diff
@@ -19,10 +19,10 @@ _<a name=spaceefficiency>**Table:**</a> `SpaceEfficiency` parameter definitions
 |Storage system type|SpaceEfficiency parameter options|
 |-------------------|---------------------------------|
 |IBM FlashSystem® A9000 and A9000R|Always includes deduplication and compression. No need to specify during configuration.|
-|IBM Spectrum® Virtualize Family|- `thick` (default value)<br />- `thin`<br />- `compressed`<br />- `deduplicated` <br /><br /> **Note:** If not specified, the default value is thick.|
-|IBM® DS8000® Family| - `none` (default value) <br />- `thin`<br /><br /> **Note:** If not specified, the default value is `none`.|
+|IBM Spectrum® Virtualize family|- `thick` (default value)<br />- `thin`<br />- `compressed`<br />- `dedup_thin` (creates volumes that are deduplicated with thin-provisioning)<br />- `dedup_compressed` (creates deduplicated and compressed volumes)<br />- `deduplicated` (creates deduplicated and compressed volumes, like `dedup_compressed`)<br /><br /> **Note:** If not specified, the default value is thick.|
```

Comment:
IMO we should also add that `deduplicated` is deprecated, to encourage the relevant users to use `dedup_compressed` instead

Reply:
DONE